tests/integration_tests/pretrained_models_test.py

```diff
@@ -102,7 +102,7 @@ def setUp(self) -> None:
         }
 
         self.coco_pretrained_maps = {
-            "ssd_lite_mobilenet_v2": 0.2052,
+            "ssd_lite_mobilenet_v2": 0.2027,
```

Comment:
Did we check the test itself ?
We use here "crowd" targets, so mAP should be slightly higher then achieved in training.

Reply:
How are we supposed to run the integration tests? Found that locally I'm unable to run these due to absence of dataset dirs. SSH to tzag and run them from cmd line is the only way?

Reply:
Ran via cmd line. It actually reached 20.41 which is reflected in test & description